regress/run_test.pl

```diff
@@ -1267,7 +1267,9 @@ sub count_postgis_objects
 sub create_db
 {
 	my $createcmd = "createdb --encoding=UTF-8 --template=template0 --lc-collate=C $DB > $REGRESS_LOG";
-	return system($createcmd);
+    my $rv = system($createcmd);
+    system("psql -d $DB -f disable_autovacuum.sql ");
```

Comment:
You can use --after-create-script switch to run arbitrary scripts. I suggest placing the script in regress/hooks/hook-after-create.sql and add the appropriate switch in core/tests.mk.in to the RUNTESTFLAGS_INTERNAL variable